homeassistant/components/sensor/ebox.py

```diff
@@ -62,25 +62,23 @@
 })
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
```

Comment:
Below, you use the new `async`/`await` syntax and here the `@asyncio.coroutine` decorator. Please convert this method to `async def async_setup_platform` and replace `yield from` with `await` in the new method.